### PR TITLE
Create plugin-lambdatest-automation.yml

### PR DESCRIPTION
**Pull request by psych0der**
# Description
Hosting request for Lambdatest automation plugin
Solved Issue: https://issues.jenkins-ci.org/browse/HOSTING-685
Target GitHub repository: https://github.com/jenkinsci/lambdatest-automation-plugin
@slide 

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by batmat** [comment]
Please check the last three checkboxes. As you can imagine, they are here for a reason, and apparently that PR is getting build failure related to these :)

**Comment by batmat** [comment]
> 
Exception in thread "main" java.lang.IllegalStateException: User name not known to Artifactory: psych0der

**Comment by psych0der** [comment]
@batmat Thanks for the clarification

File: permissions/plugin-lambdatest-automation.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "lambdatest-automation"
+github: "jenkinsci/lambdatest-automation-plugin"
+paths:
+- "org.jenkins-ci.plugins"
```
Comment (daniel-beck) [inline permissions/plugin-lambdatest-automation.yml]: This cannot possibly work. See this repo's readme.

CC @batmat

**Comment by daniel-beck** [comment]
Amended in https://github.com/jenkins-infra/repository-permissions-updater/pull/976

**Pull request merged**